frontend/src/modules/machinery/components/Machinery.css

```diff
@@ -9,29 +9,30 @@
 }
 
 .machinery > .search-wrapper {
-    border-bottom: 1px solid #5E6475;
+    border: none;
     margin: 0 10px 10px;
     position: relative;
 }
 
 .machinery > .search-wrapper .icon {
     color: #AAAAAA;
-    font-size: 19px;
-    left: 2px;
+    font-size: 17px;
+    left: auto;
     position: absolute;
-    right: auto;
+    right: 17px;
     top: 12px;
 }
 
 .machinery > .search-wrapper input[type=search] {
-    background: transparent;
-    border: none;
+    background: #272A2F;
+    border: 1px solid #4D5967;
```

Comment:
After a second look, I think we should set the background to `#4D5967`, otherwise the search boxes stand out as if they are the most important thing on Earth. :) Then you can also set `border: none;`.

Reply:
Done